### PR TITLE
New package: looking-glass-5.0.1

### PR DESCRIPTION
**Pull request by lay-by**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/looking-glass/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'looking-glass'
+pkgname=looking-glass
+version=5.0.1
+revision=1
+archs="x86_64"
```
Comment (classabbyamp) [inline srcpkgs/looking-glass/template:5]: why?

File: srcpkgs/looking-glass/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+wrksrc="$pkgname-B$version/client"
+build_style=cmake
+configure_args="-DENABLE_BACKTRACE=no"
+makedepends="cmake make gcc libglapi libglib-devel nettle-devel gmp-devel fontconfig-devel spice-devel pkg-config libX11-devel libXfixes-devel libXi-devel libXinerama-devel libXcursor-devel libXpresent-devel libxkbcommon-devel libwaylandpp libwaylandpp-devel wayland-protocols dejavu-fonts-ttf libXScrnSaver-devel MesaLib-devel"
```
Comment (classabbyamp) [inline srcpkgs/looking-glass/template:9]:
```
cmake make gcc
```
already included in the build chroot, don't specify them (gcc: by default; make: not needed, void's cmake defaults to ninja; cmake: already included by the build style)

---
```
pkg-config
```
should be in `hostmakedepends`

---
```
libglapi
```
covered by `MesaLib-devel`

---
```
libwaylandpp dejavu-fonts-ttf
```
are these actually needed at build time?

---
```
libglib-devel nettle-devel gmp-devel fontconfig-devel spice-devel ibX11-devel libXfixes-devel libXi-devel libXinerama-devel libXcursor-devel libXpresent-devel libxkbcommon-devel libwaylandpp-devel libXScrnSaver-devel MesaLib-devel
```
the rest are probably fine

File: srcpkgs/looking-glass/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+depends="dejavu-fonts-ttf"
+short_desc="Extremely low latency KVMFR for guests with VGA PCI Passthrough"
+maintainer="lay-by <bean_s@riseup.net>"
+license="GPL-2.0-or-later"
+homepage="https://looking-glass.io/"
+distfiles="$homepage/artifact/B$version/source>$pkgname-B$version.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/looking-glass/template:15]:
```suggestion
distfiles="https://looking-glass.io/artifact/B$version/source>$pkgname-$version.tar.gz"
```

Comment (lay-by) [reply srcpkgs/looking-glass/template]: I changed all of the parts you mentioned, thanks for your help and patience.

File: srcpkgs/looking-glass/template [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+build_style=cmake
+configure_args="-DENABLE_BACKTRACE=no"
+hostmakedepends="pkg-config"
+makedepends="libglib-devel nettle-devel gmp-devel fontconfig-devel spice-devel libX11-devel libXfixes-devel libXi-devel libXinerama-devel libXcursor-devel libXpresent-devel libxkbcommon-devel libwaylandpp-devel wayland-protocols libXScrnSaver-devel MesaLib-devel"
+depends="dejavu-fonts-ttf"
+short_desc="Extremely low latency KVMFR for guests with VGA PCI Passthrough"
+maintainer="lay-by <bean_s@riseup.net>"
+license="GPL-2.0-or-later"
+homepage="https://looking-glass.io/"
+distfiles="https://looking-glass.io/artifact/B$version/source>$pkgname-B$version.tar.gz"
```
Comment (JamiKettunen) [inline srcpkgs/looking-glass/template:15]: `$pkgname` -> `looking-glass`

**Pull request closed without merging**